### DISCUSSION TITLE
Treat CreateSpatialField in static index the same as CreateField

### DISCUSSION THREAD

**Discussion by BernhardGlueck** [ravendb/ravendb#17835] [Ideas]
Right now we index user defined data of several datatypes with the CreateField method ( as we don't know the fields that will be there beforehand, they are passed as a dictionary in the document ) 
One datatype that we have is a "location" ... it would be great if we could use Ravens Spatial features for this as well, and that would mean being able to use CreateSpatialField dynamically as well .

e.g 

CreateSpatialField("MyFIeldName";longitude,latitute)

**Comment by ayende** [comment]
Try using: `CreateField("MyFieldName", CreateSpatialField(long, lat))`
